lib/src/streams/stream_handler.dart

```diff
@@ -150,6 +150,8 @@ class StreamHandler extends Object with TerminatableMixin, ClosableMixin {
 
   bool get ranOutOfStreamIds => _ranOutOfStreamIds();
 
+  bool get canOpenStream => _canCreateNewStream();
```

Comment:
It'd be nice to have doc comments on new members – but since nothing else is documented...

Reply:
It's not a public api anyways, I've added a comment now, though the method name is self-describing.